app/components/Header/index.tsx

```diff
@@ -92,16 +92,17 @@ function AccountDropdownItems({
 
       <Dropdown.Divider />
       <Dropdown.ListItem>
-        <Button
-          flat
+        <Flex
```

Comment:
While this keeps the styling consistent, did you try having it as an actual button? I feel like having it as a danger-button might fit well.

Reply:
Very good point! It looks really nice as a danger button 😍 

And yes, it should have just been a `button` for it to keep some important properties such as pointer cursor etc. - I actually realized this before going to bed yesterday.